frontend/packages/ceph-storage-plugin/src/features.ts

```diff
@@ -34,9 +34,7 @@ export const RGW_FLAG = 'RGW';
 
 /* Key and Value should be same value received in CSV  */
 export const OCS_SUPPORT_FLAGS = {
-  EXTERNAL: 'EXTERNAL',
-  MINIMAL_DEPLOYMENT: 'MINIMAL_DEPLOYMENT',
-  ENCRYPTION: 'ENCRYPTION',
+  MULTUS: 'MULTUS',
```

Comment:
Can we name the flags something like `OCS-MULTUS` ? Just to differentiate with rest of the console flags.